index.bs

```diff
@@ -614,11 +614,14 @@ enum XRVisibilityState {
 [SecureContext, Exposed=Window] interface XRSession : EventTarget {
   // Attributes
   readonly attribute XRVisibilityState visibilityState;
+  readonly attribute float? frameRate;
+  readonly attribute Float32Array? supportedFrameRates;
   [SameObject] readonly attribute XRRenderState renderState;
   [SameObject] readonly attribute XRInputSourceArray inputSources;
 
   // Methods
   undefined updateRenderState(optional XRRenderStateInit state = {});
+  Promise&lt;undefined&gt; updateFrameRate(float rate);
```

Comment:
I know this was discussed earlier in the thread, but @Manishearth and I would prefer that this function be named `updateTargetFrameRate()` to disambiguate which frame rate it is setting and make it explicit that it may not be reflected in the `frameRate` attribute.